src/Sylius/Bundle/ShopBundle/Twig/Component/Checkout/Address/FormComponent.php

```diff
@@ -86,6 +96,10 @@ public function addressFieldUpdated(#[LiveArg] mixed $addressId, #[LiveArg] stri
         $newAddress['city'] = $address->getCity();
         $newAddress['postcode'] = $address->getPostcode();
 
+        foreach ($this->addressFormValuesModifiers as $modifier) {
```

Comment:
Since the `addressFormValuesModifiers` may be empty
```suggestion
        foreach ($this->addressFormValuesModifiers ?? [] as $modifier) {
```

Reply:
Fixed 👍🏻